src/test/enhetstester/kotlin/no/nav/familie/ks/sak/kjerne/beregning/endretUtbetaling/OppdaterAndelerMedEndretUtbetalingServiceTest.kt

```diff
@@ -0,0 +1,137 @@
+package no.nav.familie.ks.sak.kjerne.beregning.endretUtbetaling
+
+import no.nav.familie.ks.sak.common.util.nesteMåned
+import no.nav.familie.ks.sak.data.lagAndelTilkjentYtelse
+import no.nav.familie.ks.sak.data.lagBehandling
+import no.nav.familie.ks.sak.data.lagEndretUtbetalingAndel
+import no.nav.familie.ks.sak.data.lagPerson
+import no.nav.familie.ks.sak.data.lagPersonopplysningGrunnlag
+import no.nav.familie.ks.sak.data.randomAktør
+import no.nav.familie.ks.sak.kjerne.behandling.domene.BehandlingÅrsak
+import no.nav.familie.ks.sak.kjerne.beregning.EndretUtbetalingAndelMedAndelerTilkjentYtelse
+import no.nav.familie.ks.sak.kjerne.personopplysninggrunnlag.domene.PersonType
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import java.math.BigDecimal
+import java.time.YearMonth
+
+class OppdaterAndelerMedEndretUtbetalingServiceTest {
+    private val søker = randomAktør()
+    private val behandling = lagBehandling(opprettetÅrsak = BehandlingÅrsak.SØKNAD)
+    private val barn1 = randomAktør("01012112345")
+    private val personopplysningGrunnlag =
+        lagPersonopplysningGrunnlag(
+            behandlingId = behandling.id,
+            søkerPersonIdent = søker.aktivFødselsnummer(),
+            barnasIdenter = listOf(barn1.aktivFødselsnummer()),
+            søkerAktør = søker,
+            barnAktør = listOf(barn1),
+        )
+    private val søkerPerson = lagPerson(personopplysningGrunnlag, søker, PersonType.SØKER)
+
+    @Test
+    fun `oppdaterTilkjentYtelseMedEndretUtbetalingAndeler - endret utbetalingsandel skal overstyre andel`() {
```

Comment:
Nitpick: Om det kun er en metode som er eksponert trenger man vel ikke å prefixe det slik. Om man har flere eksponerte metoder er det som regel bedre å gruppere dem via `@Nested inner class MyMethodTest { ... }` framfor å prefixe hver enkel test med et metodenavn. Gjelder for så vidt testen under også

Reply:
Her har vi kun kopiert tester fra `TilkjentYtelseUtilTest` etter at vi splittet opp koden i 3 ulike filer. Tenkte vi ikke skulle gjøre alt for mye endringer i tillegg til omstruktureringen, men det er absolutt et godt poeng! Jeg fikser! 👍